README.md

```diff
@@ -75,6 +75,12 @@ A few tips on order subscription:
 3. Matches are price point aggregated
 4. By default markets are deleted on close
 
+## Docker
+It is possible to run the java application through a docker container:
+
+1. You may docker build and docker run or,
```

Comment:
Maybe you could specify that if someone does not want to user the compose file and build it themselves, they need to build it from the parent directory, and not from the Dockerfile directory (otherwise the paths will not work). 

You may docker build from the repository root directory (`docker build -f java/Dockerfile .`) and run or,

for example. What do you think?

Reply:
Yes Sure, will update